tests/test_models/test_huggingface.py

```diff
@@ -87,10 +54,12 @@ def test_representation_one_text_two_targets(self):
             ArtificialSubject.RecordingTarget.language_system_right_hemisphere: 'transformer.h.1.ln_1'})
         text = 'the quick brown fox'
         logging.info(f'Running {model.identifier()} with text "{text}"')
-        model.perform_neural_recording(recording_target=ArtificialSubject.RecordingTarget.language_system_left_hemisphere,
-                                       recording_type=ArtificialSubject.RecordingType.spikerate_exact)
-        model.perform_neural_recording(recording_target=ArtificialSubject.RecordingTarget.language_system_right_hemisphere,
-                                       recording_type=ArtificialSubject.RecordingType.spikerate_exact)
+        model.perform_neural_recording(
```

Comment:
are the changes just modifying whitespaces?

Reply:
yes, just newline (from autoformat, my IDE doesn't like long lines)